app/views/_layouts/nhsuk.nunjucks

```diff
@@ -27,7 +27,7 @@
     </header>
 
     <main id="content" role="main" tabindex="-1" class="util-nofocus">
-      {% block content %}{% endblock %}
+      {% block main %}{% endblock %}
```

Comment:
I'm following the convention to use namespaced names for wrapper blocks and `name_content` for inner content blocks. e.g.:

```jinja
{% block main %}
  <main id="content">
    {% block main_content %}
      <p>Some default content</p>
    {% endblock %}
  </main>
{% endblock %}
```

This allows to replace the whole container block if necessary as well as just the contents.